x-pack/legacy/plugins/apm/public/index.tsx

```diff
@@ -22,16 +24,28 @@ import { KibanaCoreContextProvider } from '../../observability/public';
 const { core } = npStart;
 
 // render APM feedback link in global help menu
-core.chrome.setHelpExtension(domElement => {
-  ReactDOM.render(
-    <KibanaCoreContextProvider core={core}>
-      <GlobalHelpExtension />
-    </KibanaCoreContextProvider>,
-    domElement
-  );
-  return () => {
-    ReactDOM.unmountComponentAtNode(domElement);
-  };
+core.chrome.setHelpExtension({
+  appName: i18n.translate('xpack.apm.feedbackMenu.appName', {
+    defaultMessage: 'APM'
+  }),
+  links: [
+    {
+      linkType: 'discuss',
+      href: 'https://discuss.elastic.co/c/apm',
+      rel: 'noopener'
```

Comment:
Would be nice if this was automatically appended to outbound links. Although that's perhaps outside the scope of this PR

Reply:
... actually: links of type `discuss` will always be outbound, correct? In that case it should be added automatically, no?

Reply:
Yeah I'm not really familiar with this property and only saw a few instance use it, so I wasn't sure if necessary. I'll look up what it does to see if it makes sense to add automatically.

Reply:
Updated to always use `noopener` on external links in help menu